pkg/routeagent/cleanuphandlers/cleanuphandlers.go

```diff
@@ -0,0 +1,6 @@
+package cleanuphandlers
+
+type CleanupHandler interface {
```

Comment:
```suggestion
type Handler interface {
```

This avoids stutter - usages will now read `cleanup.Handler.`